### DISCUSSION TITLE
LOM USI microphones in stock later this week

### DISCUSSION THREAD

**Discussion by tphakala** [tphakala/birdnet-go#284] [General]
If anyone of you are looking for a high quality and very sensitive microphone to use with BirdNET-Go etc. LOM just sent out a newsletter that their Uši series is back on stock on Friday, the 27th of September at 16:00 CEST (UTC+2).

https://store.lom.audio/collections/usi-series?mc_cid=8ac41fcbd1

As far as I know Uši is based on PUI audio AOM5024 capsule which very good for BirdNET use, I personally use these capsules for my DIY microphones. Please note that XLR models require 24-48V phantom voltage so you need a XLR audio interface, simple XLR to 3.5mm converter will not work. I recommend getting a mono microphone if you are on market for one.

**Comment by jkrauska** [comment]
seems sold out now.

**Reply by tphakala** [reply]
Yeah, those sell out very quickly - usually on the same day or the next when stock arrives.

I've been building my own microphones using PUI Audio AOM-5024 capsules. I'm working on putting together a guide on how to build high-quality microphones for BirdNET use.

**Reply by 0n3man** [reply]
@tphakala did you ever put together instructions for building that AOM-5024 microphone?  This [links](https://www.instructables.com/The-Sound-Sleuthers/) has instructions.  Is this in line with what you were going to put together?  Do you believe the 5024 capsule microphone is one of the best options?   Do you have any thoughts on how you'd make the microphone able to deal with rain and moisture?

**Comment by Shellfishgene** [comment]
I built my mic using [this simple method](https://github.com/mcguirepr89/BirdNET-Pi/discussions/39#discussioncomment-2180372) with an AOM5024. It works well, but I'm not an expert. It's been outside for a year or so, but mostly sheltered from rain. I used one of those furry wind shields instead of a foam one.
